src/pages/index.js

```diff
@@ -3,15 +3,24 @@ import Link from "next/link";
 export default function Home() {
   return (
     <div>
-      <main>
-        <h1>Accessibility Cheatsheets for Components
-        </h1>
+      <main className="home">
+        <div className="home-top">
+          <h1>Accessibility Cheatsheets for Components
+          </h1>
+          <p className="home-top__top-description">
+            Intro copy goes here, such as: Build accessible components 
+            more accurately and in less time with these easy-to-read 
+            cheatsheets based on WCAG and ARIA best practices. 
+            Check back often as we’re adding new components frequently!
+          </p>
+        </div>
         <ul>
           <li><Link href='/accordion'><a>Accordion</a></Link></li>
           <li><Link href='/dialog'><a>Dialog</a></Link></li>
           <li><Link href='/disclosure'><a>Disclosure</a></Link></li>
           <li><Link href='/tabs'><a>Tabs</a></Link></li>
         </ul>
+        <img className="home-top__top-image" src="/home-header-image.svg" alt="Illustration of dog surrounded by accessible components" />
```

Comment:
Hi Bennet, I got a linter error:
![image](https://user-images.githubusercontent.com/49820744/194418105-a76e996d-686c-413e-9504-ff7411393ef3.png)
So I think you need to use the next.js Image tag  for this

Reply:
The card does say to use img, so perhaps this is fixed in a later card? I would check with @dustin-jw

Reply:
Dustin mentioned that using the normal img element is fine, but it would be good to disable the lint warning for that one.